test/functional/test-shadow-embed.js

```diff
@@ -478,6 +478,24 @@ describes.sandboxed('shadow-embed', {}, () => {
         });
       });
     });
+
+    it('should not parse noscript as markup', () => {
```

Comment:
Can we also have a test that splits `<noscript>` and `</noscript>` in multiple writes? My concern is that I'm not sure what it'd do while parsing int he middle of `<noscript>` when the node is removed.

Reply:
Added test, seems to work fine.